modules/ROOT/pages/04-scaling.adoc

```diff
@@ -181,7 +191,7 @@ Kubernetes::
 [source,bash,subs="attributes+,+macros",linenums]
 ----
 hey -c 50 -z 10s \
-  -H "Host:prime-generator.knativetutorial.example.com" \
+  -host "prime-generator.knativetutorial.example.com" \
```

Comment:
The `hey` version I use (latest one) doesn't like specifying the host with `-H`. I needed to change that header with `-host`.